lib/smart_health_cards_test_kit/shc_payload_verification.rb

```diff
@@ -117,7 +117,7 @@ class SHCPayloadVerification < Inferno::Test
           end
         end
       end
-      output decompressed_payloads: decompressed_payload_array
+      scratch[:bundles] = fhir_bundles
```

Comment:
Why use scratch rather than something like `output fhir_bundles: fhir_bundles.to_json`?

Reply:
I think scratch should be a last resort and unless these are massive Bundles, which seems unlikely if they're coming from SHCs, I don't see a strong reason to use it here.

Reply:
OK. I thought using scratch is the first choice. I will change that.

Reply:
Changed from scratch back to input parameter with hash value.